packages/repo.rb

```diff
@@ -0,0 +1,29 @@
+
+require 'package'
+ 
+class Repo < Package
+  description 'Android Repo'
+  homepage 'https://gerrit.googlesource.com/git-repo/'
+  version '1.0'
+  license 'Apache License 2'      
+  compatibility 'x86_64' 
+  source_url 'https://storage.googleapis.com/git-repo-downloads/repo'
```

Comment:
Would it be better to use `repo` from the github mirror?

It appears to have tagged releases there.

And then you could have something like
```ruby
  version '2.32'
  license 'Apache License 2'      
  compatibility 'all' 
  source_url "https://github.com/GerritCodeReview/git-repo/raw/v#{version}/repo"
```

Reply:
Another way to do it would be to use the git functionality we have:

```ruby
  version '2.32'
  license 'Apache License 2'      
  compatibility 'all' 
  source_url "https://github.com/GerritCodeReview/git-repo.git"
  git_hashtag "v#{version}"
```
(That way, you don't need to calculate the sha256 of the file.)